.gitignore

```diff
@@ -13,4 +13,6 @@ npm-debug.log
 builderror.log
 
 bson.sublime-project
-bson.sublime-workspace
\ No newline at end of file
+bson.sublime-workspace
+
+dist
```

Comment:
Be careful with this. This will make npm ignore the dist folder when publishing, so you have to explicitly include the dist folder in the package.json

Reply:
Nice call, that seems like non ideal default behavior of npm but I do indeed see it in the docs that `.gitignore` will be used in the absence of a `.npmignore` file. I'll remove it here, but I just want to make sure that we're not committing the bundles to the git repo anymore - they should all only be packaged with the distributed tarball on npm.

Reply:
So I've confirmed that as long as the `files` array in `package.json` includes `"dist"`, it will still pack up the `dist` folder. It should be safe to add this back in.